projects/novo-elements/src/elements/data-table/configure-columns/configure-columns-dropdown.component.ts

```diff
@@ -0,0 +1,56 @@
+// NG2
+import {
+    ChangeDetectorRef,
+    Component,
+    ElementRef,
+    EventEmitter,
+    Input,
+    OnChanges,
+    Output,
+} from '@angular/core';
+// APP
+import { BasePickerResults } from '../../picker/extras/base-picker-results/BasePickerResults';
+
+@Component({
+    selector: 'configure-columns-dropdown',
+    templateUrl: './configure-columns-dropdown.html',
+})

```

Comment:
can you move the styles in here.
`styleUrls: ['./configure-columns-dropdown.scss']`. and use `:host {`